namespaces/http.yml

```diff
@@ -0,0 +1,59 @@
+version: 2016.03.10.0
+
+defaults:
+  type: string
+  doc_values: true
+  index: not_analyzed
+
+namespace:
+  name: http
```

Comment:
These are not apache logs necessarily.  The same information could be logged by apache if the services are running as WSGI applications under Apache, or could be in the service logs if the service logs are running standalone.